### DISCUSSION TITLE
Trouble loading logo to new TS101

### DISCUSSION THREAD

**Discussion by johncohn** [Ralim/IronOS#1932] [Q&A] [answered]
`First..` thanks for this awesome work. I've successfully installed IronOs on my new TS101 . 
I'm now trying to install a custom log I made using instructions [here](https://ralim.github.io/IronOS/Logo/) .. but can't get it to display on my TS101 at startup .

here's what I did:
Here is the logo I created  
![smurphysiron](https://github.com/Ralim/IronOS/assets/3111244/dc4f74ec-5d2b-485f-a52e-cd0580ba4ffd)
it is  a 96x16 bw png file (see attached) .   

I tried try the following command

`python3 img2logo.py smurphysiron.png out -m ts101`

what I get out are the following new files which don't match what I would expect given the instructions

```
johncohn@Johns-MacBook-Pro Bootup Logos % ls -tl            
total 160
-rw-r--r--   1 johncohn  staff  11550 Jun 13 09:07 outsmurphysiron_L.png.hex
-rw-r--r--   1 johncohn  staff   1333 Jun 13 09:07 outsmurphysiron_L.png.dfu
-rw-r--r--   1 johncohn  staff  11550 Jun 13 09:07 outsmurphysiron.png.hex
-rw-r--r--   1 johncohn  staff   1333 Jun 13 09:07 outsmurphysiron.png.dfu
-rw-r--r--@  1 johncohn  staff   2843 Jun 13 09:02 test.png
-rw-r--r--@  1 johncohn  staff   4161 Jun 13 07:54 smurphysiron.png
drwxr-xr-x   4 johncohn  staff    128 Jun 12 15:17 __pycache__
drwxr-xr-x@ 37 johncohn  staff   1184 Jun 12 15:12 Images
-rw-r--r--@  1 johncohn  staff   4466 Apr 10 06:53 README.md
-rwxr-xr-x@  1 johncohn  staff  13504 Apr 10 06:53 img2logo.py
-rw-r--r--@  1 johncohn  staff   1498 Apr 10 06:53 output_dfu.py
-rw-r--r--@  1 johncohn  staff   3447 Apr 10 06:53 output_hex.py
-rwxr-xr-x@  1 johncohn  staff    102 Apr 10 06:53 run.sh
```

I also tested with the -P option to create the attached 
![test.png](https://github.com/Ralim/IronOS/assets/3111244/42ab436c-53ce-4e42-bd67-2891a8ddb49c)

When I flash the resulting .hex files by moving them into the TS101 device I see mounted to my Mac (after connecting with the button near the tip pushed) , unplug, then power up with a usb charger is that  IronOs is still installed.. but I don't see my logo

I see I have two .hex files (uploaded as pdf since this forum can't upload native hex files: 
[outsmurphysiron_L.png.hex.pdf](https://github.com/user-attachments/files/15821071/outsmurphysiron_L.png.hex.pdf)
[outsmurphysiron.png.hex.pdf](https://github.com/user-attachments/files/15821072/outsmurphysiron.png.hex.pdf)

When I flash either of the  resulting .hex files by moving them into the TS101 device I see mounted to my Mac (after connecting with the button near the tip pushed) , unplug, then power up with a usb charger is that  IronOs is still installed.. but I don't see my logo

Any ideas what I might be doing wrong ? 
thanks for any help !

-jc

**Comment by ia** [comment, marked as answer]
Since there was [the same bug report already](https://github.com/Ralim/IronOS/issues/1933) with a lot of feedback and suggested advises, closing this as duplicate.
